### PR TITLE
Merge ObstacleStatus and RobotStatus

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -49,15 +49,16 @@
         target=uploader.run, name="ISAR Uploader", daemon=True
     )
     threads.append(uploader_thread)
-
+    print(settings.MQTT_ENABLED)
```
Comment (oysand) [inline main.py:52]: Remember to remove the print

File: src/isar/services/service_connections/mqtt/robot_status_publisher.py [hunk 2]
```diff
@@ -37,6 +37,8 @@ def _get_combined_robot_status(
             return RobotStatus.Offline
         elif current_state == States.Idle and robot_status == RobotStatus.Available:
             return RobotStatus.Available
+        elif current_state != States.Idle or robot_status == RobotStatus.Stuck:
```
Comment (oysand) [inline src/isar/services/service_connections/mqtt/robot_status_publisher.py:40]: Should this not be and instead of or?

Comment (oysand) [reply src/isar/services/service_connections/mqtt/robot_status_publisher.py:40]: Think we only want one catch all on the bottom, but keeping the specific once with and

**Review by oysand — CHANGES_REQUESTED** [review]
See comments

File: src/isar/services/service_connections/mqtt/robot_status_publisher.py (hunk 2 above)
Comment (UsamaEquinorAFK) [reply src/isar/services/service_connections/mqtt/robot_status_publisher.py:40]: I think we should decide this regardless of current state and make a decision based on robot_status only.

File: main.py [hunk 3]
```diff
@@ -49,15 +49,15 @@
...
     if settings.MQTT_ENABLED:
+        print("Entered MQTT Enabled block")
         mqtt_client: MqttClient = MqttClient(mqtt_queue=queues.mqtt_queue)
 
         mqtt_thread: Thread = Thread(
             target=mqtt_client.run, name="ISAR MQTT Client", daemon=True
         )
         threads.append(mqtt_thread)
-
+        print("Seeing up robot status publisher now")
```
Comment (andchiind) [inline main.py:60]: Is this meant to be "Setting"?

Comment (UsamaEquinorAFK) [reply main.py]: should not be there

**Pull request merged**